mbbsd/vote.c

```diff
@@ -519,6 +545,97 @@ vote_view_all(const char *bname)
 	return FULLUPDATE;
 }
 
+int
```

Comment:
Please put `static`, it's only used in this file. Sorry didn't mention that in the beginning. Same as other new functions you create.